array/main.js

```diff
@@ -0,0 +1,107 @@
+const log = console.log;
+let sports = ["cricket", "football", "pool", "kabaddi"]; //declare array
+log(sports[0]); // print cricket
+log(sports[4]); // undefined
+
+
+let last = sports[sports.length - 1];
+log(last); // O/P kabaddi
+
+// looping array
+sports.forEach(function (item, index, array) {
+  log(item, index); //cricket 0 football 1 pool 2 kabaddi 3
+});
+
+// add to the end of an array
+let addSport = sports.push("kho-kho");
+log(sports);
+
+// remove form the end of an array
+let lastRemove = sports.pop(); // remove kho-kho
+log(sports);
+
+// remove form the front of an array
+let firstRemove = sports.shift(); //remove cricket
+log(sports);
+
+// add to the front of an array
+let addFront = sports.unshift("cricket");
+log(sports);
+
+// find index of an item in the array
+let position = sports.indexOf("pool");
+log(position); // 2
+
+// remove an item by index position
+let removedItem = sports.splice(position, 1);
+log(removedItem); // pool
+log(sports); // 'cricket', 'football', 'kabaddi'
+
+// copy an array
+let sameCopy = sports.slice();
+log(sameCopy);
+
+// join
+log(sports.join('-')); // cricket-football-kabaddi
+log(sports.join()); // cricket,football,kabaddi
+
+// filter
+function filterItem(val) {
+  return sports.filter(function (item) {
+    return item.toLowerCase().indexOf(val.toLowerCase()) > -1;
+  });
+}
+log(filterItem('k')); // ["cricket", "kabaddi"]
+log(filterItem('oo')); // ["football"]
+
+// concat
+let equ = ["bat", "boll", "sticks"];
+const resultantArray = sports.concat(equ);
+log(resultantArray); // ["cricket", "football", "kabaddi", "bat", "boll", "sticks"]
+
+// includes
+log(resultantArray.includes('football')); // true
+log(resultantArray.includes('ket')); // false
+log(resultantArray.includes('Cricket')); // false
+
+// split
+let str = "sanket",
+  res = str.split('')

```

Comment:
Don't miss semicolons, even though JS compilers are smart in Chrome and FF, we can't trust IE.